clarifai/client/auth/abc.py

```diff
@@ -0,0 +1,23 @@
+import abc
+import grpc
+
+from clarifai_grpc.grpc.api import service_pb2_grpc
+
+
+class V2Stub(abc.ABC):
+  '''Abstract base class of clarifai api rpc client stubs'''
```

Comment:
Try to make consistent use of quotation marks throughout the docstrings. We should probably stick to """triple double quotes""" around docstrings

Reply:
Sure. Will stick to """<>"""